packages/core/admin/ee/admin/pages/SettingsPage/pages/Users/ListPage/CreateAction/index.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Button } from '@strapi/design-system/Button';
+import { Tooltip } from '@strapi/design-system/Tooltip';
+import { Icon } from '@strapi/design-system/Icon';
+import { Stack } from '@strapi/design-system';
```

Comment:
```suggestion
import { Stack, Button, Tooltip, Icon } from '@strapi/design-system';
```

Reply:
Yeah, this is actually confusing about the DS. So the correct way is to have named imports from the desing system and not the actual components? Because the FE code does it both ways and it's confusing 😅